app/javascript/flavours/glitch/features/ui/components/actions_modal.js

```diff
@@ -56,6 +56,22 @@ export default class ActionsModal extends ImmutablePureComponent {
           </a>
         </div>
 
+        <div className='favourite-modal__status-header'>
+          <div className='favourite-modal__status-time'>
+            <a href={this.props.status.get('url')} className='status__relative-time' target='_blank' rel='noopener'>
+              <RelativeTimestamp timestamp={this.props.status.get('created_at')} />
+            </a>
+          </div>
+
+          <a href={this.props.status.getIn(['account', 'url'])} className='status__display-name'>
+            <div className='status__avatar'>
+              <Avatar account={this.props.status.get('account')} size={48} />
+            </div>
+
+            <DisplayName account={this.props.status.get('account')} />
+          </a>
+        </div>
+
```

Comment:
This won't ever render because `actions_modal` is never called with a `status` prop (I'm… not entirely sure why it accepts one lol) but you don't need to duplicate this code—this would cause the status header to be rendered twice.